### PR TITLE
Refactor addresses into address line and administrative areas

### PR DESCRIPTION
**Pull request by jenssegers**
Introduce address lines and administrative areas.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -3483,19 +3483,23 @@ Delete a custom field definition
 + label: `Reception` (string, optional)
 
 ## Address (object)
-+ street: `Newell Road` (string)
-+ number: `5230` (string)
-+ postal_code: `94020` (string)
-+ city: `Palo Alto` (string)
-+ country: `US` (string)
++ line_1: `Dok Noord 3A` (string)
++ line_2: `101` (string)
```
Comment (stivni) [inline apiary.apib]: PP : I would not give this as an example...

I would:

```
+ line_1: `Dok Noord 3A/101` (string)
+ line_2: `1st floor` (string)
```

File: apiary.apib [hunk 2]
```diff
@@ -3483,19 +3483,23 @@ Delete a custom field definition
...
++ postal_code: `9000` (string)
++ city: `Ghent` (string)
++ administrative_area_2: `Oost-Vlaanderen`
```
Comment (stivni) [inline apiary.apib]: type is missing, also don't we miss optional/required here?

File: apiary.apib [hunk 3]
```diff
@@ -3483,19 +3483,23 @@ Delete a custom field definition
...
++ country: `BE` (string)
 
 ## Addressee (object)
 + addressee: `Pied Piper Finance Dep.` (string, optional)
-+ street: `Newell Road` (string)
-+ number: `5230` (string)
-+ postal_code: `94020` (string)
-+ city: `Palo Alto` (string)
-+ country: `US` (string)
++ line_1: `Dok Noord 3A` (string)
```
Comment (stivni) [inline apiary.apib]: we could use `Include Address` here

Comment (jenssegers) [reply apiary.apib]: :ok:

File: apiary.apib (hunk 2 above)
Comment (jenssegers) [reply apiary.apib]: :ok:

File: apiary.apib (hunk 1 above)
Comment (jenssegers) [reply apiary.apib]: :ok:

**Pull request merged**